ci/circle-27-compat.yaml

```diff
@@ -3,18 +3,18 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - bottleneck=1.0.0
+  - bottleneck=1.2.0
   - cython=0.28.2
   - jinja2=2.8
-  - numexpr=2.4.4 # we test that we correctly don't use an unsupported numexpr
```

Comment:
did you remove code related this this?

Reply:
Not sure where core related to this would be hiding? There was nothing in `compat.__init__`.

Reply:
```
(bamboo-dev) jreback@dev:~/pandas-dev$ more pandas/core/computation/check.py 
import warnings
from distutils.version import LooseVersion

_NUMEXPR_INSTALLED = False
_MIN_NUMEXPR_VERSION = "2.4.6"

try:
    import numexpr as ne
    ver = LooseVersion(ne.__version__)
    _NUMEXPR_INSTALLED = ver >= LooseVersion(_MIN_NUMEXPR_VERSION)

    if not _NUMEXPR_INSTALLED:
        warnings.warn(
            "The installed version of numexpr {ver} is not supported "
            "in pandas and will be not be used\nThe minimum supported "
            "version is {min_ver}\n".format(
                ver=ver, min_ver=_MIN_NUMEXPR_VERSION), UserWarning)

except ImportError:  # pragma: no cover
    pass

__all__ = ['_NUMEXPR_INSTALLED']
```